src/snowflake/connector/connection.py

```diff
@@ -192,6 +197,10 @@ def DefaultConverterClass():
         None,
         (type(None), str),
     ),  # Path to connection diag whitelist json
+    "log_imported_packages_in_telemetry": (
+        None,
+        (type(None), bool),
```

Comment:
If we are going to switch to only having a connection parameter then this should default to `True`

Reply:
done